.github/workflows/ci.yaml

```diff
@@ -33,7 +33,9 @@ jobs:
           if [[ -n "$changed" ]]; then
             echo -n "Charts changed:"
             echo "$changed"
-            echo "::set-output name=changed::$changed"
+            echo "changed<<EOF" >> $GITHUB_OUTPUT
+            echo "$changed" >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
```

Comment:
I think this will result in
```
changed
[lines changed]
```
vs `changed=[first line]`. Not 100% sure if that will work or not

Reply:
should be ok